Cargo.toml

```diff
@@ -31,7 +31,6 @@ mock-quic-p2p = { git = "https://github.com/maidsafe/quic-p2p", optional = true
 parsec = { git = "https://github.com/maidsafe/parsec" }
 quic-p2p = { version = "~0.7.0", features = ["upnp"] }
 rand = "~0.7.3"
-# rand in the versions used by the threshold-crypto crate
 rand_core = "~0.5.1"
```

Comment:
Are these differnt rand versions still required? (cargo-udeps)

Reply:
The comment was left over from last PR https://github.com/maidsafe/routing/pull/2164, when removing rand_ctypto, by mistake
rand_core is really in different version.